gtests/net/mptcp/syscalls/connect_close_full.pkt

```diff
@@ -23,6 +23,11 @@
 
 +0.0      <   .  1:1(0)  ack 1  win 450    <nop, nop,         TS val 700 ecr 100, dss dack4=2 dsn4=1 ssn=0 dll=1 nocs fin, nop, nop>
 +0.0      >   .  1:1(0)  ack 1             <nop, nop,         TS val 101 ecr 700, dss dack4=2 nocs>
++0.0      >  F.  1:1(0)  ack 1             <nop, nop,         TS val 101 ecr 700, dss dack4=2 nocs>
++0.0      <   .  1:1(0)  ack 2  win 450    <nop, nop,         TS val 700 ecr 100, dss dack4=2 nocs>
 
-// FastClose is sent
-+0.0      >  R.  1:1(0)  ack 1             <nop, nop,         TS val 101 ecr 700, mp_fastclose, mp_reset 0>
+// reply with a small delay to let the kernel switching to a time-wait socket.
```

Comment:
Could we give a bigger delay? I'm just worry about slow environments.
Half a second is maybe safer?

Reply:
Looks OK on my side. We can always tweak it later if the public CI has some issues with this.